### PR TITLE
Content updates part 1

### PR DESCRIPTION
**Pull request by sharonwarner**
contentUpdates1

Screen Shot 2020-06-24 at 4 42 07 PM

===

Resolves #XXX

- [x] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [x] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -282,28 +282,36 @@
...
+      "help-address2": "The employer's suite, floor, etc. that work was performed for this week.",
       "city": "City",
+      "help-city": "The employer's city where work was performed for this week.",
       "state": "State",
+      "help-state": "The employer's state where work was performed for this week.",
       "zipcode": "Zip",
+      "help-zipcode": "The employer's zip code where work was performed for this week.",
       "lastDateWorked": "Last Date Worked",
-      "lastDateWorkedHint": "The last day worked in this certification week",
+      "help-lastDateWorked": "The last day worked in this certification week.",
```
Comment (idealisms) [inline public/locales/en/translation.json:297]: Good catch!

File: src/client/components/EmployersQuestions/index.js [hunk 2]
```diff
@@ -62,6 +62,7 @@ function EmployerQuestions(props) {
     return (
       <Form.Group controlId={props.employerData.id + name}>
         <Form.Label>{t(tk(name))}</Form.Label>
+        <Form.Text muted>{t(tk(`help-${name}`))}</Form.Text>
```
Comment (idealisms) [inline src/client/components/EmployersQuestions/index.js:65]: Sorry for missing this!

File: src/client/components/EmployersQuestions/index.js [hunk 3]
```diff
@@ -184,18 +187,17 @@ function EmployerQuestions(props) {
...
           onChange={(e) => handleChange("reason", e.target.value)}
         >
-          {["still-working", "completed", "fired", "quit", "laid-off"].map(
-            (reason) => (
-              <option key={reason} value={reason}>
-                {t(tk("reason-" + reason))}
-              </option>
-            )
-          )}
+          {["fired", "quit", "laid-off", "still-working"].map((reason) => (
```
Comment (idealisms) [inline src/client/components/EmployersQuestions/index.js:190]: Note that the default is "still-working" (from the newEmployerData() function above). I assume that's still correct even though it's the last option?

**Comment by nguyen-tina** [comment]
Needs a comma after 4. "Union members, ..."

**Comment by nguyen-tina** [comment]
For 5., **Yes** needs to be bold

File: src/client/components/EmployersQuestions/index.js (hunk 2 above)
Comment (sharonwarner) [reply src/client/components/EmployersQuestions/index.js:65]: No problem! You didn't miss it, it was just added!

File: src/client/components/EmployersQuestions/index.js (hunk 3 above)
Comment (sharonwarner) [reply src/client/components/EmployersQuestions/index.js:190]: @nguyen-tina do you have a preference for the default option?

**Comment by nguyen-tina** [comment]
Would remove "Tell us about your income" and replace "Income 1" with "Employer 1"

**Comment by nguyen-tina** [comment]
You COULD move State and Zip to the same line, to make it less squishy

**Comment by nguyen-tina** [comment]
👀  wonder if the dropdown reasons for leaving employer has been updated?

File: public/locales/en/translation.json [hunk 4]
```diff
@@ -279,31 +279,38 @@
     "q-workOrEarn": "Did you work or earn any money, whether you were paid or not?",
     "qhelp-workOrEarn": "Look at the date each week begins and ends. If you performed any work during this week, all earnings must be reported regardless if payment has been received.",
     "employers": {
-      "p1": "Tell us about your income.",
-      "heading": "Income {{incomeNumber}}",
+      "heading": "Employer {{incomeNumber}}",
```
Comment (nguyen-tina) [inline public/locales/en/translation.json:282]: woohoo!

File: public/locales/en/translation.json [hunk 5]
```diff
@@ -227,17 +227,17 @@
     "questions": {
       "ui-full-time": {
         "tooSick": "Were you too sick or injured to work?",
-        "help-tooSick": "You must be well enough to work everyday of the week to receive full benefits. Check ‘yes’ if a workday was missed due to being sick or injured.",
-        "tooSickNumberOfDays": "If Yes, enter the number of days (1 - 7) you were unable to work.",
+        "help-tooSick": "You must be well enough to work everyday of the week to receive full benefits. Check <strong>Yes</strong> if a workday was missed due to being sick or injured.",
+        "tooSickNumberOfDays": "If yes, enter the number of days (1 - 7) you were unable to work.",
```
Comment (nguyen-tina) [inline public/locales/en/translation.json:231]: The big thing here is to not have spaces between the hyphens of the dashes and numbers ....

File: src/client/components/EmployersQuestions/index.js (hunk 3 above)
Comment (nguyen-tina) [reply src/client/components/EmployersQuestions/index.js:190]: Still working works!

File: public/locales/en/translation.json (hunk 5 above)
Comment (sharonwarner) [reply public/locales/en/translation.json:231]: Ah, I thought it was just the capitalization! Ok, that's updated!

**Pull request merged**